tests/input_dir/test_vcf_dir/test_1.vcf

```diff
@@ -25,31 +25,31 @@
 ##INFO=<ID=MQRankSum,Number=1,Type=Float,Description="Z-score From Wilcoxon rank sum test of Alt vs. Ref read mapping qualities">
 ##INFO=<ID=QD,Number=1,Type=Float,Description="Variant Confidence/Quality by Depth">
 ##INFO=<ID=ReadPosRankSum,Number=1,Type=Float,Description="Z-score from Wilcoxon rank sum test of Alt vs. Ref read position bias">
-##contig=<ID=chrM,length=16571,assembly=hg19>
```

Comment:
How does this change relate to the PR?

Reply:
I altered one of the test vcfs used for the unit tests to see if the error was fixed. When the header was formatted like: `##contig=<ID=chrM,length=16571>` the contig length returned before would be `16571>\n` instead of just `16571`. I didn't see the error prior as I was splitting on the comma, if something followed the `length` it returned it correctly. I just altered one of the test VCFs instead of adding a new one as the other one has the header formatted like: `##contig=<ID=chrM,length={LENGTH},assembly=hg38>` , so I am still testing for both cases

Reply:
As long as you understand it :) Thanks!

Reply:
These would be better tested as a non-file based unit test specifically to check all the edge cases for these fields. This way you're not potentially breaking other tests and you also have better locality of test data and the test itself.